wee_slack.py

```diff
@@ -917,6 +916,23 @@ def emoji_completion_cb(data, completion_item, current_buffer, completion):
         w.hook_completion_list_add(completion, ":" + e + ":", 0, w.WEECHAT_LIST_POS_SORT)
     return w.WEECHAT_RC_OK
 
+@utf8_decode
+def usergroups_completion_cb(data, completion_item, current_buffer, completion):
+    """
+    Adds all :-prefixed usergroups to completion list
+    """
+
+    current_channel = EVENTROUTER.weechat_controller.buffers.get(current_buffer, None)
+
+    if current_channel is None:
+        return w.WEECHAT_RC_OK
+    subteam_ids = current_channel.team.subteams.viewkeys()
+    for subteam_id in subteam_ids:
+        subteam = current_channel.team.subteams.get(subteam_id, None)
+        if subteam:
+            w.hook_completion_list_add(completion, "@" + subteam.handle, 0, w.WEECHAT_LIST_POS_SORT)
```

Comment:
Even though user groups are not nicks, I think we should set `nick_completion` to 1, because they are used like nicks. This will cause a colon to be inserted after the user group if it is completed on the start of the line.